MBBSEmu/Server/Socket/SocketServer.cs

```diff
@@ -1,4 +1,4 @@
-﻿using MBBSEmu.HostProcess;
+using MBBSEmu.HostProcess;
```

Comment:
We can remove this import as well as the _host variable/ctor if it's no longer used.